styleguide/derek/view-templates/leadership/_leadership-individual.ejs

```diff
@@ -0,0 +1,9 @@
+<% if(!className) { var className = 'leadership-individual'; } %>
+
+<div class="<%- className -%>">
+  <a href="/derek/view-templates/leadership/leadership-interior" class="leadership-attrWrapper">
+    <img src="http://placekitten.com/300/400" class="leadership-image">
+    <h4 class="leadership-name center">Mr. Mittens</h4>
+    <h6 class="leadership-title center">CEO</h6>
```

Comment:
Well then maybe we setup `leadership-titleCenter` for maintainable purposes since this is inside the pattern itself.